php/commands/site.php

```diff
@@ -1320,36 +1320,49 @@ public function service_level($args, $assoc_args) {
   * [--member=<email>]
   * : Email of the member to add. Member will receive an invite
   *
+  * [--role=<role>]
+  * : Role for the new member to act as
+  *
   * @subcommand team
   */
   public function team($args, $assoc_args) {
     $action = array_shift($args) ?: 'list';
-    $site = SiteFactory::instance(Input::sitename($assoc_args));
-    $data = array();
+    $site   = SiteFactory::instance(Input::sitename($assoc_args));
+    $data   = array();
+    $team   = $site->getUserMemberships();
     switch($action) {
       case 'add-member':
-        $team = $site->teamAddMember($assoc_args['member']);
-        Logger::coloredOutput("%2<K>Team member added!</K>");
+        $role = Input::role($assoc_args);
+        $workflow = $team->add($assoc_args['member'], $role);
+        $this->workflowOutput($workflow);
         break;
       case 'remove-member':
-        $team = $site->teamRemoveMember($assoc_args['member']);
-        Logger::coloredOutput("%2<K>Team member removed!</K>");
+        $user     = $team->findByEmail($assoc_args['member']);
+        $workflow = $user->remove($assoc_args['member']);
+        $this->workflowOutput($workflow);
+        break;
+      case 'change-role':
+        $role = Input::role($assoc_args);
+        $user = $team->findByEmail($assoc_args['member']);
+        $workflow = $user->setRole($role);
+        $this->workflowOutput($workflow);
         break;
       case 'list':
       default:
-        $team = $site->team();
-        foreach ($team as $uuid => $user) {
+        $user_memberships = $team->all();
+        foreach($user_memberships as $uuid => $user_membership) {
+          $user = $user_membership->getUser();
           $data[] = array(
-            'First' => $user->user->profile->firstname,
-            'Last'  => $user->user->profile->lastname,
-            'Email' => $user->user->email,
-            'UUID'  => $user->user->id,
+            'First' => $user->profile->firstname,
+            'Last'  => $user->profile->lastname,
+            'Email' => $user->email,
+            'UUID'  => $user->id,
```

Comment:
I've been trying to use "json" style keys for the data array that gets passed to `handleDisplay`. That seems a little more flexible for the future. But just FYI cause that isn't blocking here.